pkg/node/node.go

```diff
@@ -777,10 +777,18 @@ func NewBee(interrupt chan struct{}, sysInterrupt chan os.Signal, addr string, p
 					logger.Error(err, "unable to sync batches")
 					b.syncingStopped.Signal() // trigger shutdown in start.go
 				}
+				err = post.SetExpired()
+				if err != nil {
+					logger.Error(err, "unable to set expirations")
+				}
 			}()
 		}
 	}
 
+	if err != nil {
```

Comment:
what error are we checking here? Where is the `post.SetExpired` for full nodes ?